transforms/universal/fdedup/spark/Dockerfile

```diff
@@ -0,0 +1,56 @@
+ARG BASE_IMAGE=data-prep-kit-spark-3.5.2:0.3.0
+
+FROM ${BASE_IMAGE}
+
+# USER root
+# install pytest
+RUN pip install --no-cache-dir pytest
+
+WORKDIR ${SPARK_HOME}/work-dir
+
+# Copy in the data processing framework source/project and install it
+# This is expected to be placed in the docker context before this is run (see the make image).
+COPY --chown=spark:root data-processing-lib-python/ data-processing-lib-python/
```

Comment:
Should be getting the wheel and installing extra for ray and spark

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/6cc18cd8eaba2fb12a31f49af52aba188a9f6ac4